experimenter/experimenter/nimbus_ui_new/forms.py

```diff
@@ -106,6 +106,109 @@ def clean(self):
         return cleaned_data
 
 
+class NimbusExperimentCloneFormMixin:
```

Comment:
Yeah so technically this works but actually you probably don't even need to use a mixin here.  You can just fold this into `NimbusExperimentSidebarCloneForm` and then subclass that form for `NimbusExperimentPromoteToRolloutForm` and just override the save and changelog methods.